.travis.yml

```diff
@@ -12,10 +12,16 @@ cache:
   directories:
   - $HOME/.ivy2/cache
   - $HOME/.sbt/boot/
```

Comment:
I'm not sure that works if you don't have `language: r` but I've added a directory cache for where R is installing it's packages.